runtime/integration-tests/src/pools/utils/loans.rs

```diff
@@ -243,3 +246,42 @@ pub fn mint_nft_call(class: ClassId, instance: InstanceId, owner: AccountId) ->
 		owner: Address::Id(owner),
 	})
 }
+
+pub fn update_nav(pool_id: PoolId) -> Call {
+	Call::Loans(LoansCall::update_nav { pool_id })
+}
+
+/// Calculates the final debt for a given
+/// principal amount, rate and period.
+/// Compounding happens every second.
+///
+/// Rate should be in percent. E.g. 15 -> 15%APR
+///
+/// Logic: n = second-per-year
+/// * principal * (1 + r/n)^(end-start)
+pub fn calculate_debt(principal: Balance, rate: u64, start: Moment, end: Moment) -> Balance {
```

Comment:
I think we should try to keep using the same terminology everywhere. In the `loans` pallet we use this as naming now:

```suggestion
pub fn calculate_expected_repayment_amount(borrowed_amount: Balance, interest_rate_per_sec: u64, origination_date: Moment, maturity_date: Moment) -> Balance {
```